server/test/resolve.cpp

```diff
@@ -1108,3 +1108,106 @@ TEST_CASE("Resolve by ip address vm")
         FAIL(ex.what());
     }
 }
+
+TEST_CASE("Resolve by hosted by")
+{
+    try {
+        fty::SampleDb db(R"(
```

Comment:
@EatonEKia Do you think the test is covering all your case ?

Reply:
No, this test just for basic functionality....
I don't know (and can't imagine) all corner cases for now... if you have something, I would be glad to add it. :)

Reply:
I don't think so... this is just basic test which I can imagine... 
@EatonEKia Also I want to request more test cases.

Reply:
He will cover the VMware case where VMs are hosted on an ESXi. You will have same cases with Microsoft (microsoft.hyperv.hosts.vm) and Nutanix (nutanix.node.hosts.vm).

Reply:
But difference only in link type or something else?

Reply:
No, you're absolutly righ.